js/src/models/model.project.js

```diff
@@ -348,10 +348,19 @@ Wu.Model.Project = Wu.Model.extend({
 			title : "Could not update project", 
 			description : result.error
 		});
+
+		if(result.updated[0] == "slug") return;
+
+		// store on server
+		this.store.name = result.project.name;
+
+		// update slug name
+		!window.testMode && this.setSlug(result.project.name);
+
 		Wu.Mixin.Events.fire('projectChanged', { detail : {
 			projectUuid : this.getUuid(),
 			name : result.project.name
-		}});
+		}, projectChangedFired : true});
```

Comment:
While I was working , I saw that this event was being fired without  calling ```Wu.Mixin.Events.fire('projectChanged'```
So I tried to specify the path I want to fire. I am looking more whats the other side.